app/uk/gov/hmrc/exports/migrations/changelogs/submission/AddSubmissionFieldsForAmend.scala

```diff
@@ -34,7 +34,7 @@ class AddSubmissionFieldsForAmend extends MigrationDefinition with Logging {
       id = s"CEDS-4364 Add new fields to the Submission model for amended Declarations.",
       order = 17,
       author = "Lucio Biondi",
-      runAlways = true
+      runAlways = true // Can be removed (default is false) once this migration is deployed to production
```

Comment:
I think integration as well, because we haven't deployed there in a long time (but other migrations have also not been applied there I believe)

Reply:
Yes, yes. I just added this comment to mark the fact that the running of the migration should be disabled only after being deployed to the all envs. I will change the comment to make this more evident :+1: